sources/meta-yoe/recipes-test/fs-stress-test/fs-stress-test_git.bb

```diff
@@ -8,7 +8,7 @@ SRCREV = "354a1f6780c7c7490cda53489cd27527bee84eb6"
 PV = "git${SRCPV}"
 PR = "r1"
 
-SRC_URI = "git://github.com/cbrake/fs-stress-test.git;branch=master;protocol=git"
+SRC_URI = "git://github.com/cbrake/fs-stress-test.git;branch=master;protocol=https"
```

Comment:
Did CI catch this?

Reply:
yes, it shows up in bitbake warnings see [1]

[1] https://github.com/YoeDistro/yoe-distro/runs/4119871477?check_suite_focus=true